scripts/integration.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
```

Comment:
Hmm. I started to write an integration test in Python. I don't mind using Bash, but personally I think Python looks a million times cleaner and doesn't have those weird Bash nuances. Or we can honestly have both, the more the merrier right? Thoughts?